scripts/config.pl

```diff
@@ -54,8 +56,13 @@
 #   MBEDTLS_AES_ONLY_128_BIT_KEY_LENGTH
 #   MBEDTLS_AES_ONLY_ENCRYPT
 #   MBEDTLS_AES_SCA_COUNTERMEASURES
+#   MBEDTLS_CTR_DRBG_USE_128_BIT_KEY
```

Comment:
Why do we adding CTR_DRBG flag?

Reply:
These flags were just reordered and cleaned up.